developer/src/kmcmplib/src/kmx_u16.cpp

```diff
@@ -243,7 +243,7 @@ KMX_WCHAR * u16tok(KMX_WCHAR *p, const KMX_WCHAR ch,  KMX_WCHAR **ctx) {
   else {
     *ctx = NULL;
   }
-  return p;
+  return *p ? p : NULL;
```

Comment:
Prevents delimiters at end of string from returning an empty string. (Matching `strtok` behaviour)